tracer/src/Datadog.Trace/Ci/Ipc/CircularChannel.cs

```diff
@@ -0,0 +1,114 @@
+// <copyright file="CircularChannel.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+#nullable enable
+
+using System;
+using System.IO;
+using System.IO.MemoryMappedFiles;
+using System.Threading;
+using Datadog.Trace.Logging;
+
+namespace Datadog.Trace.Ci.Ipc;
+
+internal partial class CircularChannel : IChannel
+{
+    private const int DefaultBufferSize = 65536;
```

Comment:
Why +1? 🤔 I could understand 0 or + 4, but +1 seems odd 😅 
```suggestion
    private const int DefaultBufferSize = UInt16.MaxValue + 1;
```

Reply:
Because I was tired. 😅

Reply:
done ([dfdc51b](https://github.com/DataDog/dd-trace-dotnet/pull/5537/commits/dfdc51bc240ddca72350310470a3bfd98e88a704))